### ISSUE TITLE
Reproducing the results with Open Images

### ISSUE THREAD

**Issue by gorkem7** [qfgaohao/pytorch-ssd#102]
Thank you for this amazing repo @qfgaohao.

I am having an issue, I have downloaded 10 classes(roughly 200gb of data )with using your open_image_downloader.py script. Trained my model on that dataset, but when I introduce my model and label path in live demo script, I do not see any results. Not even a single detection. 

Here is few things:

My model name and loss is : mb2-ssd-lite-Epoch-5-Loss-3.7949399062979077.pth
I used Pytorch 1.0

training command was: python train_ssd.py --dataset_type open_images  --datasets ~data/openimage  --net mb2-ssd-lite --base_net ~Development/Mobilenetv2_SSD/models/mb2-imagenet-71_8.pth  --scheduler cosine --lr 0.01  --num_workers 8
--t_max 200 --validation_epochs 5 --num_epochs 5 --debug_steps 200

If you can help me with this, I would be appreciated. 

Thanks in advance.

**Comment by gorkem7** [comment]
Ps. I have also trained my model with 'Traffic Light' class. Nothing is happening, video is running but there is no detection.

**Comment by gorkem7** [comment]
update: Apparently I was using high prediction threshold(85%). Now, detection works but with a pretty low prediction ratio. (around 40-50)

**Comment by denizcelik** [comment]
Hi Gorkem, 

Can you say anything about my problem? Thanks.
[https://github.com/qfgaohao/pytorch-ssd/issues/120](url)
